src/components/RCSDataExtractor.h

```diff
@@ -0,0 +1,22 @@
+#include <mfem.hpp>
```

Comment:
Most of this includes are unnecessary hear. If you are not careful with includes in headers, the compile times blow up.

Reply:
TBD - Revisit old files and ensure this doesn't happen in them.